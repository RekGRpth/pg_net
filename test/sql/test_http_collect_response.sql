begin;

    with req as (
        select net.http_get('https://supabase.io', async:=true) as id
    )
    select
        net.http_collect_response(id, async:=true)
    from
        req;

rollback;
