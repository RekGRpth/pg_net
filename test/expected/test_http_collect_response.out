begin;
    with req as (
        select net.http_get('net://supabase.io', async:=true) as id
    )
    select
        net.http_collect_response(id, async:=true)
    from
        req;
 http_collect_response 
-----------------------
 
(1 row)

rollback;
/*
Since 2f88675f8df33248aa362759004850d167da6590 loading the application logs an error
"Extension Not Loaded" and does not process requests. This test must be excluded
until that is resolved or it will block forever

begin;

    with req as (
        select net.http_get('net://supabase.io', async:=true) as id
    )
    select
        net.http_collect_response(id, async:=false)
    from
        req;

r*/
